File: regress/nodejs/insert-dupe.ts
const db: ortdb = ort(dbfile);
const ctx: ortctx = db.connect();

const id1: bigint = ctx.db_foo_insert(1241);
if (id1 < 0)
	return false;

const id2: bigint = ctx.db_foo_insert(1241);
if (id2 >= 0)
	return false;

return true;

File: regress/rust/insert-dupe.rs
use orb::ort;
use std::env;

fn main() {
    let args: Vec<String> = env::args().collect();
    assert_eq!(args.len(), 2);

    let ctx = ort::connect(&args[1]).unwrap();

    let id1 = ctx.db_foo_insert(100).unwrap();
    assert_ne!(id1, -1);
    let id2 = ctx.db_foo_insert(100).unwrap();
    assert_eq!(id2, -1);
}
